webapp/components/admin_console/login_settings.jsx

```diff
@@ -74,6 +139,103 @@ export default class LoginSettings extends AdminSettings {
 
         return (
             <SettingsGroup>
+                <TextSetting
+                    id='passwordMinimumLength'
+                    label={
+                        <FormattedMessage
+                            id='admin.password.minimumLength'
+                            defaultMessage='Minimum Password Length:'
+                        />
+                    }
+                    helpText={
+                        <FormattedMessage
+                            id='admin.password.minimumLengthDescription'
+                            defaultMessage='Minimum number of characters required for a valid password.'
+                        />
+                    }
+                    value={this.state.passwordMinimumLength}
+                    onChange={this.handleChange}
+                />
+                <Setting
```

Comment:
I couldn't find `handleChange` in any of the other components, but moving it out is a good idea. The commit should be in soon